framework/Simulation.py

```diff
@@ -149,236 +146,8 @@ def splitCommand(s):
     retList.append(buffer)
   return retList
 
-def createAndRunQSUB(simulation):
```

Comment:
Yes, it was PBS Pro specific, so it has been moved to part of the custom mode classes.